config/karma-docs.conf.js

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
@ThomasBurleson This file is new.  We should discuss offline what the strategy is for testing the docs site.  There were no specs prior to this PR.